### PR TITLE
Rendering Updates

### PR DESCRIPTION
**Pull request by ggada**
Code, image rendering fix
Fix exports for for multi-level bulleted and ordered lists in org-mode.

### CODE REVIEW

File: xwiki.lua [hunk 1]
```diff
@@ -22,7 +22,7 @@
 -- syntax errors.
 
 -- Character escaping
--- I don't think we need any excaping.
+-- I don't think we need any escaping.
```
Comment (krupan) [inline xwiki.lua:25]: Good catch!

File: xwiki.lua [hunk 2]
```diff
@@ -54,10 +54,13 @@ end
...
 -- Table to store footnotes, so they can be included at the end.
 local notes = {}
+local bullets = {}
+local orders = {}
 
 -- Blocksep is used to separate block elements.
 function Blocksep()
-  return "\n\n"
+  return bullets[1] and "\n" or orders[1] and "\n" or "\n\n"
+--  return "\n\n"
```
Comment (krupan) [inline xwiki.lua]: Just delete code, don't comment it out.  There are several other places where that can be done, I won't call them all out individually

**Review by krupan — COMMENTED** [review]
I don't have an xwiki instance to test with right now, so I'll trust you on the changes.  Just clean up the commented out code like I mentioned and I'll merge this.

File: xwiki.lua (hunk 1 above)
Comment (ggada) [reply xwiki.lua:25]: :)

File: xwiki.lua (hunk 2 above)
Comment (ggada) [reply xwiki.lua]: Done.

**Review by krupan — APPROVED** [review]

**Comment by krupan** [comment]
Thanks!

**Pull request merged**